binaries/fpgad/api/opae_events_api.c

```diff
@@ -71,8 +72,8 @@ int opae_api_register_event(int conn_socket,
 
 STATIC void release_event_registry(api_client_event_registry *r)
 {
-	close(r->fd);
-	free(r);
+	opae_close(r->fd);
+	opae_free(r);
```

Comment:
Are these resources acquired with an opae_wrapper in a different file?

Reply:
The fd comes to fpgad via the special events socket.  The r pointer is from opae_malloc() in opae_api_register_event() in the same file.